pkgs/development/python-modules/environmental-override/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "environmental-override";
+  version = "0.1.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1vhd37i6f8xh6kd61yxc2ynzgcln7v2p7fyzjmhbkdnws6gwfs6s";
+  };
+
```

Comment:
Are there tests we can run? If not please disable running checks.

Please add a pythonImportsCheck.

Reply:
There aren't any tests, so I've set `doCheck` to `false`.

I've added a pythonImportsCheck.